openff/qcsubmit/workflow_components/base.py

```diff
@@ -145,21 +152,22 @@ def list_components() -> List[Components]:
 
 # register the current components
 # conformer generation
-register_component(StandardConformerGenerator())
```

Comment:
🎉

Reply:
Thought you might like this, thanks for showing me another way to do it in bespokefit!